src/Symfony/Component/Workflow/Dumper/DumperHelperTrait.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Dumper;
+
+trait DumperHelperTrait
```

Comment:
I would remove this trait and inline the code.

Reply:
I removed it.